QUICKSTART.md

```diff
@@ -0,0 +1,354 @@
+  - [a. Stand up two kind clusters: florin and guilder](#a-stand-up-two-kind-clusters-florin-and-guilder)
+  - [b. Create a KubeStellar Inventory Management Workspace (IMW) and Workload Management Workspace (WMW)](#b-create-a-kubestellar-inventory-management-workspace-imw-and-workload-management-workspace-wmw)
+  - [c. Onboarding the clusters](#c-onboarding-the-clusters)
+  - [d. Create and deploy the Apache Server workload into florin and guilder clusters](#e-create-and-deploy-the-apache-server-workload-into-florin-and-guilder-clusters)
+- [3. Teardown the environment](#3-teardown-the-environment)
+
+
+This guide is intended to show how to quickly bring up a **KubeStellar** environment with its dependencies from a binary release.
+
+## 1. Install and run **KubeStellar**
+
+KubeStellar works in the context of kcp, so to use KubeStellar you also need kcp. Download the kcp and **KubeStellar** binaries and scripts into a `kubestellar` subfolder in your current working directory using the following command:
+
+```shell
+bash <(curl -s https://raw.githubusercontent.com/kcp-dev/edge-mc/main/hack/bootstrap-kubestellar.sh) --kcp-version v0.11.0 --kubestellar-version v0.1.0 --folder .
+export PATH="$PATH:$(pwd)/kcp/bin:$(pwd)/kubestellar/bin"
+export KUBECONFIG="$(pwd)/.kcp/admin.kubeconfig"
+```
+
+Check that `KubeStellar` is running:
+
+First, check that controllers are running with the following command:
+
+```shell
+ps aux | grep -e mailbox-controller -e placement-translator -e scheduler
+```
+
+which should yield something like:
+
+```console
+user     1892  0.0  0.3 747644 29628 pts/1    Sl   10:51   0:00 mailbox-controller --inventory-context=root --mbws-context=base -v=2
+user     1902  0.3  0.3 743652 27504 pts/1    Sl   10:51   0:02 scheduler -v 2 --root-user kcp-admin --root-cluster root --sysadm-context system:admin --sysadm-user shard-admin
+user     1912  0.3  0.5 760428 41660 pts/1    Sl   10:51   0:02 placement-translator --allclusters-context system:admin -v=2
+```
+
+Second, check that the Edge Service Provider Workspace (`espw`) is created with the following command:
+
+```shell
+kubectl ws tree
+```
+
+which should yield:
+
+```console
+kubectl ws tree
+.
+└── root
+    ├── compute
+    └── espw
+```
+
+## 2. Example deployment of Apache HTTP Server workload into two local kind clusters
+
+### a. Stand up two kind clusters: florin and guilder
+
+Create the first edge cluster:
+
+```shell
+kind create cluster --name florin --config examples/florin-config.yaml
+```  
+
+Create the second edge cluster:
+
+```shell
+kind create cluster --name guilder --config examples/guilder-config.yaml
+```  
+
+### b. Create a KubeStellar Inventory Management Workspace (IMW) and Workload Management Workspace (WMW)
+
+IMWs are used by KubeStellar to store inventory objects (`SyncTargets` and `Locations`). Create an IMW named `example-imw` with the following command:
```

Comment:
What follows is not a command, it is three commands.  The plural form should be used here, and in the WMW case.

Reply:
Fixed

Reply:
resolving